conda-recipes/fairseq-feedstock/ci_support/build_steps.sh

```diff
@@ -0,0 +1,62 @@
+# (C) Copyright IBM Corp. 2018, 2019. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+export PYTHONUNBUFFERED=1
+export FEEDSTOCK_ROOT=/home/conda/feedstock_root
+export RECIPE_ROOT=/home/conda/recipe_root
+export CI_SUPPORT=/home/conda/feedstock_root/.ci_support
+export CONFIG_FILE="${CI_SUPPORT}/${CONFIG}.yaml"
+
+if [[ "$CONFIG" == "linux_python3.6" ]]; then
```

Comment:
This should include 3.7 case too. Right? I mean x86 specific binary should be downloaded for both 3.6 and 3.7 so, linux_python3.6 and linux_python3.7 both should be covered in this condition. 
As per current code, even for linux_python3.7, it would go into else part and download ppc64le specific Anaconda.

Reply:
yes, this would be needed. Currently using 1.6.1 code so had not added it. But eventually it would be needed, so will just add :-)